### DISCUSSION TITLE
How to Make the Hightlight of a List View Disappear

### DISCUSSION THREAD

**Discussion by eeriemyxi** [Textualize/textual#4087] [General]
![image](https://github.com/Textualize/textual/assets/86738916/ea166d72-836f-4c0a-bf06-e707b33e5b4a)

How to make the blue highlight disappear from the list view? It is ruining my theming and is of no use in my program as it is fully prompt-dependent.

I apologize if the query is _too general,_ but I don't use Discord, so I assumed this is the only alternative.

**Comment by davep** [comment]
You'd want to be styling `ListView > ListItem.--highlight` and `ListView:focus > ListItem.--highlight`; although it does then raise the question of how the user will know which item has the highlight; and if the reason is that you don't want the user to know, that would then suggest that `ListView` isn't the widget you want in the first place.

Perhaps [it might make more sense to say what you want to achieve here](https://label.dev/articles/xy-problem/); perhaps there's a more appropriate approach to take?

**Reply by eeriemyxi** [reply]
Thanks.

> Perhaps [it might make more sense to say what you want to achieve here](https://label.dev/articles/xy-problem/); perhaps there's a more appropriate approach to take?

Basically, other to-do apps for desktop don't meet my desires, so I am currently trying to pull a quick one out myself. I already have a functioning implementation, but some of my planned essential features are not officially supposed to be supported using `rich` module that I was utilizing, so I trying to migrate to Textual in a reckless manner. Other built-in widgets were not very visually appealing by default, so I picked some CSS off the example snippets along with ListView because they seemed to have everything I was looking for, excluding the highlighter.

The green text that you are seeing is each task's generated ID and all the commands depend on them. Due to the nature of all the commands, I don't really have an efficient use of the highlighting:

![image](https://github.com/Textualize/textual/assets/86738916/bb175209-9e8a-4dc8-b062-9e490acc76ea)

**Reply by davep** [reply]
So if I'm to understand you, you want to be able to place widgets in a vertical scrolling container that doesn't do highlighting and doesn't need any form of direct interaction because all changes will be done from a command-input widget? If so `ListView` would seem to do more than you need.

You probably want to use one of the [containers](https://textual.textualize.io/api/containers/) and [mount and remove widgets](https://textual.textualize.io/tutorial/#dynamic-widgets) within it.

**Reply by eeriemyxi** [reply]
![image](https://github.com/Textualize/textual/assets/86738916/91451432-7a01-4b73-a20a-30c51dcd0b64)

Thanks. I made it work with containers, `HorizontalScroll` and `VerticalScroll`. I should style them later using TCSS I guess.

**Reply by davep** [reply]
Glad you're finding your way; feel free to ask if you need clarification of anything in the docs.
